examples/example-app/README.md

```diff
@@ -0,0 +1,8 @@
+### Build project
+```
```

Comment:
Let's put a nice title and a few sentences about what the example does so they don't have to parse through code and comments to figure out.